### PR TITLE
Update kuttl tests for clowdjobs

### PR DESCRIPTION
**Pull request by BlakeHolifield**
* Fixes unsupported event type "Error"

### CODE REVIEW

**Comment by psav** [comment]
Can you revert the label changes here :) As we reverted :p

**Review by psav — CHANGES_REQUESTED** [review]
Please revert the change to labels

**Review by psav — CHANGES_REQUESTED** [review]
Let's take out the timeout and we can merge this

**Comment by psav** [comment]
@BlakeHolifield We do need the sleep 90 there however :p

**Review by psav — APPROVED** [review]

File: controllers/cloud.redhat.com/clowdjobinvocation_controller.go [hunk 1]
```diff
@@ -135,7 +135,7 @@ func (r *ClowdJobInvocationReconciler) Reconcile(req ctrl.Request) (ctrl.Result,
 		// Match the crd.Job name to the JobTemplate in ClowdApp
 		job, err := getJobFromName(jobName, &app)
 		if err != nil {
-			r.Recorder.Eventf(&app, "Error", "JobNameMissing", "ClowdApp [%s] has no job named", cji.Spec.AppName, jobName)
+			r.Recorder.Eventf(&app, "Warning", "JobNameMissing", "ClowdApp [%s] has no job named", cji.Spec.AppName, jobName)
```
Comment (psav) [inline controllers/cloud.redhat.com/clowdjobinvocation_controller.go:138]: I just realised - I think you are passing more arguments here than your format string knows how to decode.

**Review by psav — CHANGES_REQUESTED** [review]

File: controllers/cloud.redhat.com/clowdjobinvocation_controller.go (hunk 1 above)
Comment (BlakeHolifield) [reply controllers/cloud.redhat.com/clowdjobinvocation_controller.go:138]: .... you are correct.

**Review by psav — APPROVED** [review]

**Pull request merged**